packages/mds-api-server/api-server.ts

```diff
@@ -40,13 +40,13 @@ export const ApiServer = (
 
   // Middleware
   app.use(
-    PrometheusMiddleware(options.prometheus),
     CompressionMiddleware(options.compression),
     RequestLoggingMiddleware(options.requestLogging),
     CorsMiddleware(options.cors),
     JsonBodyParserMiddleware(options.jsonBodyParser),
     MaintenanceModeMiddleware(options.maintenanceMode),
-    AuthorizationMiddleware(options.authorization)
+    AuthorizationMiddleware(options.authorization),
+    PrometheusMiddleware(options.prometheus)
```

Comment:
I assume this moving to the end of the middleware chain is intentional? If so, it may be worth commenting or even placing it in a separate `.use`.

Reply:
Might be worth moving the `.use(PrometheusMiddleware(options.prometheus))` after the `/health` route

Reply:
Yes, I moved it to the end because it needs to be after the Authorization middleware so it can grab claims in the label transformers. I'm up for either putting it in its own `.use` or adding a comment